spec/models/conversion_host/configurations_spec.rb

```diff
@@ -21,14 +21,16 @@
         }
       end
       it "to succeed and send notification" do
-        allow_any_instance_of(ConversionHost).to receive(:enable_conversion_host_role)
+        allow(ConversionHost).to receive(:new).and_return(conversion_host)
```

Comment:
Since you're doing this multiple times, you can move it into a `before` block.

Reply:
nice!